README.rst

```diff
@@ -14,25 +14,49 @@
 .. image:: https://img.shields.io/github/license/reanahub/reana-ui.svg
    :target: https://github.com/reanahub/reana-ui/blob/master/LICENSE
 
+.. image:: https://img.shields.io/badge/code_style-prettier-ff69b4.svg
+   :target: https://github.com/prettier/prettier
+
 About
------
+=====
 
 REANA-UI is a component of the `REANA <http://www.reana.io/>`_ reusable and
 reproducible research data analysis platform. REANA-UI provides a web interface
 to review production and historical workflows.
 
 Features
---------
+========
+
+- profile page containing REANA access token
+- list of personal workflows
+- workflow details page containing logs, files, specification
+- GitLab integration to load your workflow repositories
+
+Quick start
```

Comment:
I feel this `Quick start` before the `Usage` might be confusing, what do you think about adding it after and calling it something like `Development server`/`Development` or similar?